src/Console/Command/GenerateFromCommand.php

```diff
@@ -17,63 +17,62 @@
 class GenerateFromCommand extends Command
 {
     // the name of the command (the part after "bin/console")
-    const OPTION_INTO_FILE_STORAGE = "into-file-storage";
-    const OPTION_REWRITE = "rewrite";
+    private const OPTION_INTO_FILE_STORAGE = "into-file-storage";
+
+    private const OPTION_REWRITE = "rewrite";
+
     protected static $defaultName = 'generate-from';
 
-    protected function configure()
+    protected function configure(): void
     {
-        $this->addArgument("folder", InputArgument::REQUIRED);
-        $this->addOption(self::OPTION_INTO_FILE_STORAGE, 's', InputOption::VALUE_NEGATABLE, 'Generate into yaml file', false);
-        $this->addOption(self::OPTION_REWRITE, 'r', InputOption::VALUE_NEGATABLE, '', false);
-        $this->setDescription("...");
+        $this
```

Comment:
4 lines vs 15 lines. You don't need to read this code as often, because everything will be displayed in help section of the command. I think it is redundant